src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -87,6 +91,11 @@ const BioAndDMSetting: React.FC<BioAndDMSettingProps> = ({
   };
 
   const toggleSwitchAnonAllowFollowing = () => {
+    if (isAllowFollowingSendDM) {
+      setIsAllowFollowingSendDM(() => false);
+      return;
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic here seems to disable the switch if it's currently enabled, without any further checks. This seems fine as long as it's the intended behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->